package.json

```diff
@@ -45,6 +45,9 @@
 		"typescript": "^4.4.3"
 	},
 	"dependencies": {
-		"@octokit/core": "^3.5.1"
+		"@octokit/core": "^3.5.1",
+		"@octokit/rest": "^18.12.0",
+		"@vscode/test-electron": "^2.1.2",
+		"github-graphql-api": "^1.0.6"
```

Comment:
I distrust this package. Non-official, no maintenance for a while...
https://www.npmjs.com/package/github-graphql-api

Reply:
Oh yeah we're not using that at the very end, let me remove it